### DISCUSSION TITLE
Enable Copy / Paste of Actions across Widgets

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7485] [Ideas]
Scenario 

I have a nice screen with a super duper button that does multiple things. Now, the time came, that i want to move some of that functionality, or all, over to an Icon cause the super duper Button is too big for Mobile. 
It would be nice to be able to copy the Actions of the Button and paste them over to the Icon than having to recreate them all again.

**Comment by melohagan** [comment]
That's a really cool idea. I think in general it would be nice to change the component type to any compatible type.

E.g.
- Button -> Icon
- Data Provider -> Repeater

**Reply by poirazis** [reply]
Yes, that would be nice too!
